app/lambda/lambda_function.py

```diff
@@ -1,29 +1,39 @@
 import boto3
 import os
 import json
+from get_credentials.lambda_function import GetCredentialsLambda
+from create_health_care_professional.lambda_function import CreateHealthCareProfessionalLambda
+from create_patient.lambda_function import CreatePatientLambda
+from list_health_care_professionals.lambda_function import ListHealthCareProfessionalsLambda
+from list_patients.lambda_function import ListPatientsLambda
+from create_session.lambda_function import CreateSessionLambda
+from list_sessions.lambda_function import ListSessionsLambda
 
-def lambda_handler(context, event):
-    print("event: {}".format(event))
-    sts = boto3.client('sts' , 
-    endpoint_url = 'https://sts.{}.amazonaws.com'.format(os.environ['AWS_REGION'])
-    )
-    transcribeCredentials = sts.assume_role(
-        RoleArn=os.environ['TRANSCRIBE_ACCESS_ROLEARN'],
-        RoleSessionName="access_transcribe_role"
-    )['Credentials']
-    result = {}
-    result['accessKeyId'] = transcribeCredentials['AccessKeyId']
-    result['secretAccessKey'] = transcribeCredentials['SecretAccessKey']
-    result['sessionToken'] = transcribeCredentials['SessionToken']
-    result['region'] = os.environ['AWS_REGION']
-    return {
-        "isBase64Encoded": False,
-        "statusCode": 200,
-        'body': json.dumps(result),
-        "headers": {
-            'Content-Type': 'application/json',
-            'Access-Control-Allow-Origin': '*',
-            'Access-Control-Allow-Headers': '*',
-            'Access-Control-Allow-Methods': 'GET,POST,OPTIONS'
-            }
-    }
+def lambda_handler(event, context):
+    options = {'/getCredentials': GetCredentialsLambda(),
+               '/createHealthCareProfessional': CreateHealthCareProfessionalLambda(),
+               '/createPatient': CreatePatientLambda(),
+               '/listHealthCareProfessionals': ListHealthCareProfessionalsLambda(),
+               '/listPatients': ListPatientsLambda(),
+               '/createSession': CreateSessionLambda(),
+               '/listSessions': ListSessionsLambda()}
+    if event['resource'] in options:
+        return options[event['resource']].handle(event, context)
+    else:
+        raise Exception("operations not supported")
```

Comment:
Please rephrase to "Operation not support" Since at a single time, only one API( or one operation) is being called so keeping the exception to a single operation (and not operations) is ideal

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/691348b78d89ba8652fa177e9a002b4466b0ef6b/app/lambda/lambda_function.py#L22